CountlyConnectionManager.m

```diff
@@ -764,19 +784,27 @@ - (NSData *)pictureUploadDataForQueryString:(NSString *)queryString
     NSString* boundaryStart = [NSString stringWithFormat:@"--%@\r\n", kCountlyUploadBoundary];
     NSString* contentDisposition = [NSString stringWithFormat:@"Content-Disposition: form-data; name=\"pictureFile\"; filename=\"%@\"\r\n", localPicturePath.lastPathComponent];
     NSString* contentType = [NSString stringWithFormat:@"Content-Type: image/%@\r\n\r\n", allowedFileTypes[fileExtIndex]];
-    NSString* boundaryEnd = [NSString stringWithFormat:@"\r\n--%@--\r\n", kCountlyUploadBoundary];
 
     NSMutableData* uploadData = NSMutableData.new;
     [uploadData appendData:[boundaryStart cly_dataUTF8]];
     [uploadData appendData:[contentDisposition cly_dataUTF8]];
     [uploadData appendData:[contentType cly_dataUTF8]];
     [uploadData appendData:imageData];
-    [uploadData appendData:[boundaryEnd cly_dataUTF8]];
     return uploadData;
 #endif
     return nil;
 }
 
+- (void)addMultipart:(NSMutableData *)uploadData andKey:(NSString *)key andValue:(NSString *)value
+{
+    NSString* boundaryStart = [NSString stringWithFormat:@"\r\n--%@\r\n", kCountlyUploadBoundary];
```

Comment:
I am not sure about it but is it a right way to add boundary start on every Multipart and then only one boundary end for all of them?